index.js

```diff
@@ -45,10 +45,10 @@ function setTempo(tempo) {
 
 function setSequence(division, sequence) {
   if (typeof division !== 'number') throw new TypeError('Division must be a number');
-  if (!(sequence instanceof Array)) throw new TypeError('Sequence must be an array');
+  if (sequence && !(sequence instanceof Array)) throw new TypeError('Sequence must be an array');
 
   this.division = division;
-  this.sequence = sequence;
+  this.sequence = sequence || [];
```

Comment:
Sure, we could make the sequence optional. But maybe we should default to an Array that is the same length as the number of subdivisions (or maybe 16, typical step-sequencer length)
```javascript
this.sequence = sequence || Array(divisions);
// or
this.sequence = sequence || Array(16);
```